tenable/io/v3/base/schema/explore/utils.py

```diff
@@ -0,0 +1,32 @@
+'''
+Utils for explore schema
+'''
+
+from tenable.errors import ValidationError
+from tenable.io.v3.base.schema.explore.search import SortSchema
+from typing import Dict, List, Tuple
+
+
+def generate_sort_data(kw: Dict, is_with_prop: bool = True) -> List[Dict]:
+    schema = SortSchema()
+    sort_data = []
+    data = kw.get('sort')
+    if isinstance(data, List):
+        for tup in data:
+            if isinstance(tup, Tuple) and len(tup) == 2:
+                property = tup[0]
+                order = tup[1]
+                data_validate = dict(property=property, order=order)
+                data_validate = schema.load(data_validate)
+                if is_with_prop:
+                    sort_data.append(schema.dump(data_validate))
+                else:
+                    sort_data.append({
+                        property: order
+                    })
+            else:
+                raise ValidationError('Sort', '[("field_name_1", "asc"), ("field_name_2", "desc")]')
```

Comment:
It seems that you've guarded a good bit in the condition above, but I'm not completely certain that it'll cover everything. Let's say someone enters:
```
[("field_name_1", 0), ("field_name_2", 1)]
```
Are we handling that error case some other way?

Reply:
@tomeara yes we made sure that only these two options will get passed from the schema else it will be blocked.
![image](https://user-images.githubusercontent.com/93529282/144088025-161c1c92-9eaa-4d49-82f7-640c37fc16e0.png)

Reply:
Ah, sorry I missed that. Sounds great!